src/Authentication/Timestamp.php

```diff
@@ -48,7 +48,11 @@ public function isValid()
         $now = new \DateTime('now', new \DateTimeZone('UTC'));
         $timestamp = clone $this->timestamp;
 
-        return $timestamp->add(new \DateInterval($this->validFor)) >= $now;
+        try {
+            return $timestamp->add(new \DateInterval($this->validFor)) >= $now;
+        } catch (\Exception $e) {
```

Comment:
Can you be more specific on the exceptions here?

Reply:
Yes, the exception is to handle invalid interval_spec. 
http://php.net/manual/en/dateinterval.construct.php

Reply:
I meant the type of exception being caught; but I now see that it _is_ `\Exception`, so this is good!